verifier/src/test/groovy/com/okta/jwt/impl/OktaJWTClaimsVerifierTest.groovy

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2017 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.jwt.impl
+
+import com.nimbusds.jose.JOSEException
+import com.nimbusds.jose.proc.SimpleSecurityContext
+import com.nimbusds.jwt.JWTClaimsSet
+import com.nimbusds.jwt.proc.BadJWTException
+import com.okta.jwt.JoseException
+import com.okta.jwt.JwtVerifier
+import com.okta.jwt.TestSupport
+import org.testng.annotations.Test
+
+import java.security.NoSuchAlgorithmException
+import java.time.Instant
+
+class OktaJWTClaimsVerifierTest {
+
+    private int clockSkewOffset = 65 * 1000 // default skew is 60 seconds
+
+    private final def basicValidClaims = [
+            exp: Instant.now().epochSecond + 10000,
```

Comment:
They better not be! If so I'm going to have a long chat with my intellij settings.

That said, I been noticing that intellij adds an extra indent when a statement spans multiple lines.